modules/group_alert_banner/group_alert_banner.module

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/**
+ * @file
+ * Various hook implementations.
+ */
+
+use Drupal\localgov_alert_banner\Entity\AlertBannerEntityTypeInterface;
+
+/**
+ * Implements hook_ENTITY_TYPE_insert().
+ *
+ * This is necessary during configuration imports.  This makes the plugin
+ * definitions from this module immediately available to other config files.
+ */
+function group_alert_banner_localgov_alert_banner_type_insert(AlertBannerEntityTypeInterface $banner_type) {
+
+  \Drupal::service('group_relation_type.manager')->clearCachedDefinitions();
+}
+
+/**
+ * Implements hook_localgov_microsites_roles_default().
+ *
+ * Defines default site-wide and group-specific Alert banner permissions.
+ *
+ * @see Drupal\localgov_microsites_group\RolesHelper::getModuleRoles()
+ */
+function group_alert_banner_localgov_microsites_roles_default() {
+
+  $alert_banner_related_role_permissions = [
+    // phpcs:disable Drupal.Classes.FullyQualifiedNamespace
+    'global' => [
+      Drupal\localgov_microsites_group\RolesHelper::MICROSITES_CONTROLLER_ROLE => [
+        'access localgov alert banner listing page',
+        'use localgov_alert_banners transition create_new_draft',
+        'use localgov_alert_banners transition publish',
+        'use localgov_alert_banners transition unpublish',
+        'view all localgov alert banner entities',
+        'view all localgov alert banner entity pages',
+      ],
+      Drupal\localgov_microsites_group\RolesHelper::MICROSITES_EDITOR_ROLE => [
+        'access localgov alert banner listing page',
+        'use localgov_alert_banners transition create_new_draft',
+        'use localgov_alert_banners transition publish',
+        'use localgov_alert_banners transition unpublish',
+        'view all localgov alert banner entities',
+        'view all localgov alert banner entity pages',
+      ],
+    ],
```

Comment:
Granting this to the editors would be a major change in the permisions model, which is that alerts are by the emergency publisher. 
(Although I'm aware of councils that change this to also allow editors).
Is the emergency publisher role not something avalible on microsites?